btf/btf_test.go

```diff
@@ -377,3 +377,58 @@ func TestTypesIterator(t *testing.T) {
 		t.Fatal("Cannot find 'iphdr' type")
 	}
 }
+
+func TestLoadSplitSpecFromReader(t *testing.T) {
+	spec, err := LoadSpecFromReader(readVMLinux(t))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	f, err := os.Open("testdata/xt_nat")
```

Comment:
This is fine for now, but we should replace this with split BTF for `bpf_testmod` from one of the kernels we use on CI. See https://github.com/cilium/ebpf/commit/544f053c951ed3b02a881a4fe349ec7203a99225

Reply:
Are you suggesting instead of having the copy of kmod BTF (e.g., `xt_nat`) to load the `bpf_testmod`, and then to read the split BTF from `/sys/kernel/btf/bpf_testmod`? If yes, then my worry is that it will not be so easy to test the pkg `btf` with just `go test ./btf`.